src/components/StampOverlay/StampOverlay.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../constants/styles';
+@import '../../constants/overlay';
+
+.StampOverlay {
+  @extend %overlay;
+
+  top: 46px;
+  width: 232px;
+  border-radius: 4px 0 0 4px;
+  box-shadow: 0 2px 16px 0 #E1E1E3;
+  width: 354px;
+  padding: 16px 16px 16px;
+
+
+  .rubber-stamp {
+    float: left;
+    // background: #f8f8f8;
+    // border-radius: 4px 4px 4px 4px;
+    height: 48px;
+    width: 160px;
+    // margin-bottom: 8px;
+
+    img {
+      width: 100%;
+      height: 100%;
+      cursor: pointer;
+    }
+  }
+  .rubber-stamp:hover {
+    // background: antiquewhite;
+    background: var(--button-hover-color);
```

Comment:
Does css variable work in IE11?

Reply:
I changed to 
```
background: #F6F6F6;
background: var(--button-hover-color);
```
so it can fallback to otherone is `var` is not supported
it is used the same way as in Button.scss